client/client-benchmark-noop-api-plugin/build.gradle

```diff
@@ -20,5 +20,4 @@ esplugin {
 tasks.named("assemble").configure { enabled = false }
 tasks.named("dependencyLicenses").configure { enabled = false }
 tasks.named("dependenciesInfo").configure { enabled = false }
-// no unit tests
-tasks.named("test").configure { enabled = false }
+tasks.named("uploadSnykDependencyGraph").configure { enabled = false }
```

Comment:
this plugin applies `elasticsearch.build` but I don't consider it production code